net/grpcurl/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                grpcurl
+version             1.3.0
+categories          net
+platforms           darwin
+supported_archs     x86_64
+license             MIT
+
+maintainers         @gearnode
+
+checksums           rmd160 479a66bb0634e2d15b92321fcf14454224403128 \
+                    sha256 b800d949ed1184b276bf1d5471211dd60a204e5199db0725efb59a4a09166221
+
+homepage            https://github.com/fullstorydev/grpcurl
```

Comment:
Please use the GitHub portgroup.

Reply:
@mf2k do you documentation link for github port group?

Reply:
https://guide.macports.org/chunked/reference.portgroup.html

or just look at any other portfile in the repo that also uses this PG. there are many.